bazel/gen_compilation_database.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 
 #TODO(lizan): revert to released version once new version is released
-RELEASE_VERSION=d5a0ee259aa356886618eafae17ca05ebf79d6c2
+#TODO(keith): revert after https://github.com/grailbio/bazel-compilation-database/pull/30 is resolved
+RELEASE_VERSION=e65a17bef735ef64457221800207f6c295a6c8aa
 
 if [[ ! -d bazel-compilation-database-${RELEASE_VERSION} ]]; then
-  curl -L https://github.com/grailbio/bazel-compilation-database/archive/${RELEASE_VERSION}.tar.gz | tar -xz
+  curl -L https://github.com/keith/bazel-compilation-database/archive/${RELEASE_VERSION}.tar.gz | tar -xz
```

Comment:
Generally dislike pointing to private forks for many reasons (security, auditability, maintainability), but since this is just for the compilation database (no impact on production Envoy binaries) and will be fixed soon, plus you are well known to the maintainers I'm going to merge. CC @envoyproxy/maintainers FTR, as we should keep a trail of this sort of thing.

Reply:
Actually, it looks like that PR is likely to land soon, so maybe we can hold back a day or so and avoid this.

Reply:
Yea I just pingged that PR asking it to be merged, I didn't intend we merge with this.

Reply:
My PR merged there, so I've changed this back to an upstream sha.